openstates/wi/__init__.py

```diff
@@ -134,6 +134,7 @@
                       'events', 
                       'influenceexplorer'],
     '_ignored_scraped_sessions': [
+        'January 2017 Special Session',
```

Comment:
Done. If you'd rather we put it in as a session (which works, but it returns no bills) i'm fine with that. 

This is kind of a weird one, it ended a while ago so i'm not sure if they're just late getting the data or never going to add it. 

I'll try to remember to keep an eye on http://docs.legis.wisconsin.gov/2017/proposals/JR7/ to see if they backfill it.